centos_ci.sh

```diff
@@ -374,7 +374,7 @@ else
     perform_nightly "b2d";
   elif [[ "$JOB_NAME" = "minishift-nightly-minikube" ]]; then
     perform_nightly "minikube";
-  elif [[ "$JOB_NAME" = "minishift-nightly-centos" ]]; then
+  elif [[ "$JOB_NAME" = "minishift-pr" ]]; then
```

Comment:
??? PR performs nightly ?

Reply:
Just for testing purpose in CentOS ISO where it is affecting